src/node/platform.cpp

```diff
@@ -58,24 +58,8 @@ void copy_bundled_realm_files()
 
 void remove_realm_files_from_directory(const std::string &dir_path)
 {
-    DIR *dir = opendir(dir_path.c_str());
-    if (!dir) {
-        return;
-    }
-
-    while (struct dirent *entry = readdir(dir)) {
-        if (strstr(entry->d_name, ".realm")) {
-            // Intentionally not complaining if there was an error.
-            if (entry->d_type == DT_DIR) {
-                rmdir(entry->d_name);
-            }
-            else {
-                unlink(entry->d_name);
-            }
-        }
-    }
-
-    closedir(dir);
+    std::string delete_realms = "rm -rf " + dir_path + "/*.realm*";
+    system(delete_realms.c_str());
```

Comment:
I am still against adding this, but if you absolutely insist on not just writing the recursive delete, then please at least single quote the `dir_path` like this: `"rm -rf '" + dir_path + "'/*realm*"`